ServerStatisticsCollector.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Newtonsoft.Json;
+using ServerMonitoringSystem;
+using ServerMonitoringSystem.RabbitMQMessage;
+using System.Diagnostics;
+using Weather_Monitoring.ReadConfig;
+public class ServerStatisticsCollector
+{
+    private readonly IMessageQueue _messageQueue;
+
+    public ServerStatisticsCollector(IMessageQueue messageQueue)
+    {
+        _messageQueue = messageQueue;
+    }
+
+    public async Task StartCollecting()
+    {
+        var samplingIntervalSeconds = ConfigReader.ReadConfig().samplingIntervalSeconds;
```

Comment:
shouldn't depend on a static class nor concrete implementation here, that would make it hard to test the class.